scripts/md2rst.py

```diff
@@ -29,5 +29,5 @@ def download_pandoc():
 def md2rst(text):
     download_pandoc()
     output = pypandoc.convert_text(text, 'rst', format='md',
-     filters=[os.path.join('filters', 'yml_filter.py')])
+     filters=[os.path.join(os.getcwd(),'filters', 'yml_filter.py')])
```

Comment:
The current working directory when the script is run is the `yaydoctemp` dir. The `script` directory is copied inside as part of the build process. The path to `yml_filter.py` should be `scripts/filters/yml_filter.py`. I would suggest to replace `os.getcwd()` with `'scripts'`.